src/frontends/pytorch/src/utils.cpp

```diff
@@ -555,6 +555,25 @@ Output<Node> masked_fill(ov::pass::NodeRegistry& rg,
     return rg.make<opset10::Select>(bool_mask, _value, data);
 }
 
+Output<Node> concat_dims_to_shape(const NodeContext& context, size_t begin, size_t end) {
```

Comment:
Naming is not great. It can be used not only for shapes, but for any list inputs, I propose name `concat_list_from_inputs`

Reply:
Updated the naming